src/sidebar/util/test/thread-test.js

```diff
@@ -0,0 +1,36 @@
+import * as threadUtil from '../thread';
+
+describe('sidebar/util/thread', () => {
+  let fakeThread = {
```

Comment:
I whipped this up quickly but am going to keep an eye on it in case we need to manage test fixtures that represent threads more formally.